setup.cfg

```
@@ -1,2 +1,3 @@
 [flake8]
-ignore = E501
\ No newline at end of file
+ignore = E501
```

Can you add a comment saying this ignores errors where line length is > 70 something chars?